website/content/docs/enterprise/license/overview.mdx

```diff
@@ -0,0 +1,69 @@
+---
+layout: docs
+page_title: Consul Enterprise License
+description: Consul Enterprise License Overview.
+---
+
+# Consul Enterprise License
+
+## Licensing
+
+All Consul Enterprise agents must be licensed when they are started. Where that license comes from will depend
+on which binary is in use, whether the agent is a server, client or snapshot agent and whether ACLs have been
+enabled for the cluster.
+
+-> ** Consul Enterprise v1.10.0 removed temporary licensing.** In previous versions Consul Enterprise
```

Comment:
I don't think we typically prefix the version with `v`. Could be wrong.

Reply:
Removed all `v`